machineconfiguration/v1/generated.pb.go

```diff
@@ -0,0 +1,18223 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: github.com/openshift/api/machineconfiguration/v1/generated.proto
+
+package v1
+
+import (
+	fmt "fmt"
+
+	io "io"
+
+	proto "github.com/gogo/protobuf/proto"
+	github_com_gogo_protobuf_sortkeys "github.com/gogo/protobuf/sortkeys"
+	github_com_openshift_api_config_v1 "github.com/openshift/api/config/v1"
+	k8s_io_api_core_v1 "k8s.io/api/core/v1"
+	v11 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+
+	math "math"
+	math_bits "math/bits"
+	reflect "reflect"
+	strings "strings"
+
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+func (m *AWSPlatformSpec) Reset()      { *m = AWSPlatformSpec{} }
```

Comment:
Confused bc this AWSPlatformSpec should come from "github.com/openshift/api/config/v1"

Reply:
Not sure why it's erroring in it's own generated file that was built using `make generate-with-container` but here was are again.